projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -90,6 +90,14 @@
 	font-weight: 700;
 }
 
+:where( .grunion-checkbox-multiple-options legend ),
+:where( .grunion-radio-options legend ) {
```

Comment:
What's the rationale for reducing to zero? In general I think it should also be aiming for 0-1-0 for consistency, otherwise themes have a hard time working around it.

Reply:
No rationale. I'm playing Whac-A-Mole. 😄 

> 0-1-0 for consistency

I'll fix this, thanks for confirming!